src/mtxLora/commands/uplink/GetHalfhoursEnergies.ts

```diff
@@ -118,6 +150,26 @@ class GetHalfhoursEnergies extends Command {
             ...buffer.getBytesToOffset()
         ]);
     }
+
+    toJson ( {dlms}: IDlmsJsonOptions = defaultDlmsJsonOptions ) {
+        const {parameters} = this;
+        const {
```

Comment:
it looks very sparse but gives not very much

Reply:
fixed